shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -2455,7 +2467,10 @@ private void updateRecursively(
         }
       }
 
+      int previousNodeId = -1;
```

Comment:
Would be good to leave a comment explaining why traversal order of the child list alone is not sufficient on Android.